src/consts.ts

```diff
@@ -35,9 +35,9 @@ export const TipsThresholdSmall = 750;
 export const TipsThresholdMedium = 1200;
 
 /*
- * Toggle-able services
+ * Toggle-able plugins
```

Comment:
we don't need the "toggle-able" terminology anymore.

Just put "Available plugins".